src/darwin/Framework/CHIP/templates/CHIPClustersObjc-src.zapt

```diff
@@ -31,8 +31,22 @@ using namespace chip::app::Clusters;
 {{#*inline "callbackName"}}{{#if hasSpecificResponse}}{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase responseName}}{{else}}CommandSuccess{{/if}}{{/inline}}
 - (void){{asLowerCamelCase name}}With{{#if (hasArguments)}}Params:(CHIP{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase name}}Params * {{#unless (commandHasRequiredField .)}}_Nullable{{/unless}})params completionHandler{{else}}CompletionHandler{{/if}}:({{>command_completion_type command=.}})completionHandler
 {
+  CHIPCommandArguments * arguments;
+  {{#if mustUseTimedInvoke}}
+  arguments = [CHIPCommandArguments new];
+  arguments.timedInvokeTimeoutMs = @10000;
+  {{/if}}
+  [self {{asLowerCamelCase name}}WithArguments: arguments{{#if (hasArguments)}} params:params completionHandler{{else}} completionHandler{{/if}}:completionHandler];
+}
+- (void){{asLowerCamelCase name}}WithArguments: (CHIPCommandArguments * _Nullable)arguments{{#if (hasArguments)}} params:(CHIP{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase name}}Params * {{#unless (commandHasRequiredField .)}}_Nullable{{/unless}})params completionHandler{{else}} completionHandler{{/if}}:({{>command_completion_type command=.}})completionHandler
```

Comment:
So what I was assuming we would do is make the thing that has the timedInvokeTimeout bit a super-class for the command params structs.  That has the concern that if some command sometime defines a field called timedInvokeTimeoutMs we will have a name collision, but that seems pretty unlikely at first glance...

Reply:
I tried this out, but it is possible that some commands may not have arguments. When the command does not have arguments, the command params struct will not be available and we will have no way of passing down timedInvokeTimeoutMs.

Reply:
My point is we should always generated the command params structs, and for the ones where there are no arguments have a signature that takes no params (and defaults things) and one that takes the params struct.

Reply:
Ah I see. I updated the PR. Thank you.